LiskHQ/lisk-sdk
# Protocol-spec second signature - closes #4148

### What was the problem?

- Genesis block was being used and was requested to be removed
- No protocol spec generators for account's second signature existed

### How did I solve it?

By adding the required generators and its JSON output

### How to manually test it?

- Build should be green 
- run `protocol-specs/index.js` and check output in `generator_outputs/block_processing_second_signature`

### Review checklist

- [x] The PR resolves #4148
